packages/iTwinUI-react/src/core/NotificationMarker/NotificationMarker.tsx

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { CommonProps, useTheme } from '../utils';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/notification-marker.css';
+
+export type NotificationMarkerProps = {
+  /**
+   * Content of the NotificationMarker.
+   */
+  children: React.ReactNode;
+  /**
+   * Type of notification
+   *
+   * 'primary' = blue,
+   * 'positive' = green,
+   * 'warning' = orange,
+   * 'negative' = red,
+   * @default 'primary'
+   */
+  type?: 'primary' | 'positive' | 'warning' | 'negative';
+  /**
+   * Set this to true for important notifications
+   * @default false
+   */
+  urgent?: boolean;
+  /**
+   * Set this programmatically to false when you just want to render the passed children without the notification
+   * @default true
+   * @example
+   * let [newMessagesCount, ...] = useState(0);
+   * ...
+   * <NotificationMarker active={newMessagesCount > 0}>
+   *   <SvgNotification />
+   * </NotificationMarker>
+   */
+  active?: boolean;
+} & Omit<CommonProps, 'title'>;
+
+/**
+ * A small notification circle to the top-right of the passed children prop. This can be applied to pretty much anything but mostly intended for icons within default / borderless buttons.
+ * @example
+ * // Primary Intended Use-case
```

Comment:
No need for this `//` comment

Reply:
To confirm, did you mean to remove line 46? Or just remove the "`//`" from line 46?

I thought you meant the former, so I removed line 46. But just confirming if that's what you meant.